pkgs/development/libraries/libtorrent-rasterbar/default.nix

```diff
@@ -1,26 +1,32 @@
-{ stdenv, fetchurl, fetchpatch, automake, autoconf, boost, openssl, lib, libtool, pkgconfig, zlib, python, libiconv, geoip, ... }:
+{ stdenv, lib, fetchFromGitHub, fetchpatch, pkgconfig, automake, autoconf, zlib
+, boost, openssl, libtool, python, libiconv, geoip }:
 
 stdenv.mkDerivation rec {
   name = "libtorrent-rasterbar-${version}";
   version = "1.1.7";
+  formattedVersion = lib.replaceChars ["."] ["_"] version;
```

Comment:
I am (almost always) a fan of having less (garbage/temporary) attributes inside the actual `mkDerivation` attrset. While we are already touching the expression in some parts. How about you change it to something like:

```nix
{…}:
let # the `let` expression is always recursive per default
  version = "1.1.7";
  formattedVersion = lib.replaceChars ["."] ["_"] version;
in
stdenv.mkDerivation { # note the missing `rec` here
  name = "libtorrent-rasterbar-${version}";
  src = fetchFromGitHub { … };
  …
}
```